Apps/WebClient/src/ClientApp/src/views/profile.vue

```diff
@@ -78,6 +79,15 @@ export default class ProfileView extends Vue {
     @Action("addError", { namespace: "errorBanner" })
     addError!: (error: BannerError) => void;
 
+    @Action("getPatientData", { namespace: "user" })
```

Comment:
It seems you don't need this action and the isPatientLoading getter.

Reply:
Added the getter to the mounted and removed the isPatienLoading

Reply:
Looks good to me.